Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM golang:1.15 as builder
+WORKDIR /go/src/github.com/gojekfarm/stevedore
+COPY ./ ./
+RUN make compile
+
+FROM alpine:latest
+ENV HELM_VERSION="v3.4.0"
```

Comment:
Does the helm version need to be same as the one used by stevedore?
https://github.com/gojekfarm/stevedore/blob/master/go.mod#L84

Reply:
Yes, it should be the same as the one used by stevedore

Reply:
Changed to v3.3.1 (which is same as go.mod version) @jskswamy @arunvelsriram